src/containers/shared/components/Transaction/EscrowCancel/Description.tsx

```diff
@@ -9,12 +10,12 @@ import { TRANSACTION_ROUTE } from '../../../../App/routes'
 import { RouteLink } from '../../../routing'
 
 const Description: TransactionDescriptionComponent = (
-  props: TransactionDescriptionProps,
+  props: TransactionDescriptionProps<EscrowCancel>,
 ) => {
   const { t, i18n } = useTranslation()
   const language = i18n.resolvedLanguage
   const { data } = props
-  const deleted: any = findNode(data, 'DeletedNode', 'Escrow')
+  const deleted: any = findNode(data.meta, 'DeletedNode', 'Escrow')
```

Comment:
Shouldn't this be done in the parser?

Reply:
For some reason the Description components never imported the parser are had the result of the parser passed in. I think that could be done in another PR because it would apply to almost every Description component.

Reply:
Created #778